my-react-app/src/app/screens/Login/components/CustomInput/index.js

```diff
@@ -0,0 +1,18 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+export const CustomInput = ({ label, input, meta, type }) => (
+  <Fragment>
+    <label htmlFor={input.name}> {label} </label>
```

Comment:
Don´t leave spaces between the tags and the content.
`<label htmlFor={input.name}> {label} </label>`
`<label htmlFor={input.name}>{label}</label>`